netbox/dcim/views.py

```diff
@@ -2579,12 +2579,18 @@ def get_extra_context(self, request, instance):
             data=vlans,
             orderable=False
         )
+        vlan_translation_table = InterfaceVLANTranslationTable(
```

Comment:
It might be cleaner to just set `vlan_translation_table = None` if `instance.vlan_translation_policy` is null (and update the template accordingly).